examples/minimal/Feast 101.ipynb

```diff
@@ -126,13 +85,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "CORE_HOST = os.getenv(\"DEMO_FEAST_CORE_SERVICE_HOST\", \"localhost\")\n",
-    "SERVING_HOST = os.getenv(\"DEMO_FEAST_ONLINE_SERVING_SERVICE_HOST\", \"localhost\")\n",
-    "REDIS_HOST = os.getenv('DEMO_REDIS_MASTER_SERVICE_HOST', 'localhost')\n",
+    "REDIS_HOST = os.getenv('DEMO_REDIS_MASTER_SERVICE_HOST', os.getenv('FEAST_REDIS_HOST'))\n",
+    "DEMO_DATA_LOCATION = os.getenv(\"DEMO_DATA_LOCATION\")\n",
```

Comment:
Should we rename this from `demo` to something more generic? demo seems specific to our demos whereas this notebook is version controlled for use as a tutorial.

Reply:
I have already moved the demo variable in https://github.com/feast-dev/feast/pull/1086.

So perhaps we can #1086  merged first?